src/exampleTypes/CheckableTermsStatsTable.js

```diff
@@ -0,0 +1,35 @@
+import _ from 'lodash/fp'
+import F from 'futil-js'
+import React from 'react'
+import { observer } from 'mobx-react'
+import { Column } from '../layout/ExpandableTable'
+import TermsStatsTable from './TermsStatsTable'
+
+let CheckableTermsStatsTable = observer(
+  ({ children, Checkbox, getValue, selected, ...props }) => (
+    <TermsStatsTable
+      {...{
+        ...props,
+        children: [
+          <Column
```

Comment:
If the intent is to have a checkbox on each row, I don't think this needs to use the Column's expand object, you can just move the display function as a prop on the Column. Unless I am missing something on how it should work?